types/stripe/index.d.ts

```diff
@@ -194,6 +195,12 @@ declare namespace Stripe {
              * This field is null unless business_type is set to individual.
              */
             individual?: IIndividual;
+
+             /**
+              * External accounts (bank accounts and debit cards) currently
+              * attached to this account
+              */
+            external_accounts: IList<cards.ICard | bankAccounts.IBankAccount>;
```

Comment:
Should should be marked optional, since it's only available to Custom and Express accounts.

Reply:
ok I will make that change